### ISSUE TITLE
Update old policy with new policy in PPO

### ISSUE THREAD

**Issue by azzurras18** [sarcturus00/Tidy-Reinforcement-learning#3]
Hi, I'm looking at this code https://github.com/sarcturus00/Tidy-Reinforcement-learning/blob/master/Algorithm/PPO_pendulum.py but I don't understand where it is the part of code that allow to update old policy with new policy

**Comment by sarcturus** [comment]
... in "def __init__"
 self.action = actor.choose_action(self.OBS)
self.neglogp = actor.get_neglogp(self.OBS, self.ACT, reuse=True)
ratio = self.neglogp / self.NEGLOGP
clip_ratio = tf.clip_by_value(ratio, 1. - self.clip_range, 1. + self.clip_range)
actor_loss = -tf.reduce_mean((tf.minimum(ratio, clip_ratio)) * self.ADV)

... and in "def learn"
[self.sess.run(self.actor_train_op, {self.OBS: obs, self.ACT: act, self.ADV: adv, self.NEGLOGP: neglogp}) for _ in range(10)]

Thanks for asking questions. I've been doing other things lately and I have not been able to get an answer fast. You have to read the paper well, the update part is not as intuitive as the policy gradient or DQN
